### PR TITLE
Python update

### PR DESCRIPTION
**Pull request by dturanski**
Updates CLI to use Python 3 and cleans up samples

### CODE REVIEW

File: samples/python/upper/README.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
+= square
```
Comment (trisberg) [inline samples/python/upper/README.adoc]: should be "upper" - can fix on merge

File: samples/python/concat/README.adoc [hunk 2]
```diff
@@ -0,0 +1,31 @@
+= square
```
Comment (trisberg) [inline samples/python/concat/README.adoc]: Will change to "concat" on merge

Comment (dturanski) [reply samples/python/concat/README.adoc]: Thanks I missed a spot

**Comment by trisberg** [comment]
Squashed/Rebased merged as commit b20ff0264908b96f99d2b136819c5b90ea463843 and commit aff04abf09b5a0d086212f1d1b63e974bf18343d

**Pull request closed without merging**